### PR TITLE
test: Add fedora-25 images

### PR DESCRIPTION
**Pull request by stefwalter**
Removing fedora-23 will be in a separate pull request, along with other updates.

### CODE REVIEW

**Comment by stefwalter** [comment]
Nothing related to storaged seems to work on Fedora 25. The UI says that the storaged API isn't available.

**Comment by stefwalter** [comment]
Blocked on #5031

**Comment by mvollmer** [comment]
> Nothing related to storaged seems to work on Fedora 25. The UI says that the storaged API isn't available.

SELinux. :-/

**Comment by mvollmer** [comment]
Apart from the SELinus issues, this also needs #5032.

**Comment by mvollmer** [comment]
> SELinux. :-/

https://bugzilla.redhat.com/show_bug.cgi?id=1375156

**Comment by stefwalter** [comment]
Rebased and running tests again, will add naughty files once we get a full view of what's failing again.

**Comment by stefwalter** [comment]
Added more known issues.

**Comment by stefwalter** [comment]
Added two more known issues.

File: test/README [hunk 1]
```diff
@@ -76,12 +76,13 @@ You can set these environment variables to configure the test suite:
...
                 "fedora-23"
                 "fedora-24"
+                "fedora-25"
                 "fedora-testing"
                 "fedora-atomic"
                 "centos-7"
                 "rhel-7"
                 "debian-unstable"
-             "fedora-24" is the default (testinfra.py, testlib.sh)
+             "fedora-25" is the default (testinfra.py, testlib.sh)
```
Comment (dperpeet) [inline test/README]: If that is so, it should be marked as such in [testinfra.py:49](https://github.com/cockpit-project/cockpit/pull/5018/files#diff-6eef19f6e12cab1b25ceb2f3d8765caaR49)

Comment (dperpeet) [inline test/README]: Also, testlib.sh is now testlib.py, but that now gets its defaults properly from testinfra

**Comment by dperpeet** [comment]
I could fix this during merge, if the proposed change is ok.

**Comment by stefwalter** [comment]
Fine with fixing during merge.

Sad that fedora-25 is such a mess. But this is the reality: On Fedora new versions are targetting 25 ... and that's the version that should be our testing focus.

**Pull request closed without merging**